File: Java/src/testEgencia/MainPaiment.java
package testEgencia;

import java.util.List;

public class MainPaiment {
    public static void main(String[] args) {
        CreditCard creditCard = new CreditCard();
        PayPal payPal = new PayPal();

        List<PaymentMode> paymentModeList = List.of(creditCard, payPal);

    }
}

File: Java/kotlin/testEgencia/MainPaiment.kt
package kotlin.testEgencia

object MainPaiment {
    @JvmStatic
    fun main(args: Array<String>) {
        val creditCard = CreditCard()
        val payPal = PayPal()

        val paymentModeList = listOf<PaymentMode>(creditCard, payPal)
    }
}

